tests/integration/template_complete_run.cfg

```diff
@@ -25,9 +25,9 @@ years = "1850:1854:2", "1850:1854:4",
   vars = "PRECT"
 
 [ts]
-active = True
+active = 4
 walltime = "00:30:00"
-years = "1850:1854:2",
+years = "1850:1854:2"
```

Comment:
If `active = True` above, then the error we get here is `ValueError: Error interpreting years 1`. `climo`'s multiple year sets above no longer appear to need a trailing comma. However, this error seems to indicate at least one comma must be present in a `string_list` parameter.